package-res/resources/web/pentaho/lang/Filter.js

```diff
@@ -44,64 +85,64 @@ define([
   });
 
 
-  var Tree = Node.extend({
-    get type() { return "tree";},
-    constructor: function() {
-      this._children = [];
+  var AbstractTreeFilter = AbstractFilter.extend({
+    constructor: function(children) {
+      this._children = children ? children.slice() : [];
```

Comment:
Yes. All classes could accept a single element or an array of.
Each element is converted to a Filter individually.